cmake/EkatCreateUnitTest.cmake

```diff
@@ -289,12 +289,10 @@ function(EkatCreateUnitTestFromExec test_name test_exec)
 
       # Setup valgrind/memcheck commmand modifications
       if (EKAT_ENABLE_VALGRIND)
-        if (EKAT_ENABLE_MPI)
-          set(VALGRIND_SUP_FILE "${CMAKE_BINARY_DIR}/mpi.supp")
-        else()
-          set(VALGRIND_SUP_FILE "${CMAKE_BINARY_DIR}/serial.supp")
+        if (NOT EKAT_VALGRIND_SUPPRESSION_FILE)
```

Comment:
Why not resorting to the ones we generate (mpi.supp or serial.supp) if this is not present? It

Reply:
@bartgol , that should be how it works. EKAT_VALGRIND_SUPPRESSION_FILE is FORCE-set by the valgrind_support CMake file if it's not already set.

Reply:
Ah, I missed the FORCE.